src/main/java/network/brightspots/rcv/ContestConfigMigration.java

```diff
@@ -188,6 +188,11 @@ static void migrateConfigVersion(ContestConfig config)
         }
       }
 
+      // New rule to stop tabulation early
```

Comment:
Nit: unnecessary comment.

Reply:
I updated the comment to clarify that this is for a 1.3.0 to 1.4.0 migration, since I expect there will be additional migrations and it would be nice to keep them organized in case we eventually create a more full-featured migration system.

Reply:
Cool, that seems more useful to me.